### PR TITLE
[NetAppFiles] Update to api-version 2023-05-01

### PR DESCRIPTION
**Pull request by audunn**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by audunn** [comment]
@live1206 Can we please have this pr looked at to move things along.

File: sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # Release History
 
 ## 1.4.0-beta.1 (Unreleased)
-
+- Updated to support ANF api-version 2022-09-01
```
Comment (live1206) [inline sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md:4]: This same line has been added for 1.1.0 and 1.3.0.
If this intends to describe the api-verison, should it be 2023-05-01 for the current release?

File: sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
 # Release History
 
 ## 1.4.0-beta.1 (Unreleased)
```
Comment (live1206) [inline sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md:3]: Please fill in the expecting release date.

File: sdk/netapp/Azure.ResourceManager.NetApp/tests/ResourceTests/NetAppResource_NetworkSiblingSetTests.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.NetApp.Tests
+{
+    public class NetAppResource_NetworkSiblingSetTests: NetAppTestBase
+    {
+        private NetAppAccountCollection _netAppAccountCollection { get => _resourceGroup.GetNetAppAccounts(); }
+        private readonly string _pool1Name = "pool1";
+        //public static new AzureLocation DefaultLocation = AzureLocation.WestUS2;
```
Comment (live1206) [inline sdk/netapp/Azure.ResourceManager.NetApp/tests/ResourceTests/NetAppResource_NetworkSiblingSetTests.cs:21]: consider to remove the commented line if not needed any more.

File: sdk/netapp/Azure.ResourceManager.NetApp/tests/ResourceTests/NetAppResource_NetworkSiblingSetTests.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+                    foreach (NetAppVolumeResource volume in volumeList)
+                    {
+                        await volume.DeleteAsync(WaitUntil.Completed);
+                    }
+                    await LiveDelay(30000);
+                    await pool.DeleteAsync(WaitUntil.Completed);
+                }
+                await LiveDelay(40000);
+                //remove
+                //await _capacityPool.DeleteAsync(WaitUntil.Completed);
```
Comment (live1206) [inline sdk/netapp/Azure.ResourceManager.NetApp/tests/ResourceTests/NetAppResource_NetworkSiblingSetTests.cs:68]: can these be removed?

**Comment by audunn** [comment]
@live1206 , @ArthurMa1978 Updated based on comments, thanks

**Comment by audunn** [comment]
@live1206 , @ArthurMa1978 After syncing with main I now get this when re-running a test (to reslove pipeline issue)
Any idea what can cause this ?

```
  Failed UpdateNetAppAccount [45 ms]
  Error Message:
   System.UriFormatException : Invalid URI: The format of the URI could not be determined.
  Stack Trace:
     at System.Uri.CreateThis(String uri, Boolean dontEscape, UriKind uriKind)
   at Azure.ResourceManager.TestFramework.ManagementRecordedTestBase`1.GetArmEnvironment(TestEnvironment environment) in C:\Users\audunn\Source\repos\azure\azure-sdk-for-net\common\ManagementTestShared\Redesign\ManagementRecordedTestBase.cs:line 147
   at Azure.ResourceManager.TestFramework.ManagementRecordedTestBase`1.GetCleanupClient() in C:\Users\audunn\Source\repos\azure\azure-sdk-for-net\common\ManagementTestShared\Redesign\ManagementRecordedTestBase.cs:line 108
   at Azure.ResourceManager.TestFramework.ManagementRecordedTestBase`1.CreateCleanupClient() in C:\Users\audunn\Source\repos\azure\azure-sdk-for-net\common\ManagementTestShared\Redesign\ManagementRecordedTestBase.cs:line 167
```

**Review by live1206 — APPROVED** [review]

**Comment by audunn** [comment]
@live1206 Thank you, when can we expect this to be released?

**Comment by live1206** [comment]
> @live1206 Thank you, when can we expect this to be released?

It will be 10/15 as you set in the changelog.

**Comment by live1206** [comment]
> @live1206 I see we have put 1.4.0-beta.1 as version in the changelog. perhaps we should change that to 1.4.0 as this is based on stable version?

It's OK to release a beta version based on stable swagger, but not the other way around.
If you want to release stable version, please raise a new PR to update the version in changelog and csproj file to stable version instead. I will release it tomorrow for you.
Sorry that I forgot to release it for you, but it's good in this way.

**Pull request merged**